src/cli/monitor.ts

```diff
@@ -42,51 +42,54 @@ export default function (program: RootCmd) {
       const log = new K8sLog(kc);
 
       const logStream = new stream.PassThrough();
-
       logStream.on("data", chunk => {
         const respMsg = `"msg":"Check response"`;
         // Split the chunk into lines
         const lines = chunk.toString().split("\n");
-
-        for (const line of lines) {
-          // Check for `"msg":"Hello Pepr"`
-          if (line.includes(respMsg)) {
-            try {
-              const payload = JSON.parse(line);
-              const isMutate = payload.res.patchType || payload.res.warnings;
-
-              const name = `${payload.namespace}${payload.name}`;
-              const uid = payload.uid;
-
-              if (isMutate) {
-                const plainPatch = atob(payload.res.patch) || "";
-                const patch = JSON.stringify(JSON.parse(plainPatch), null, 2);
-
-                const patchType = payload.res.patchType || payload.res.warnings || "";
-
-                const allowOrDeny = payload.res.allowed ? "🔀" : "🚫";
-                console.log(`\n${allowOrDeny}  MUTATE     ${name} (${uid})`);
-                if (patchType.length > 0) {
-                  console.log(`\n\u001b[1;34m${patch}\u001b[0m`);
-                }
-              } else {
-                const failures = Array.isArray(payload.res) ? payload.res : [payload.res];
-
-                const filteredFailures = failures
-                  .filter((r: ResponseItem) => !r.allowed)
-                  .map((r: ResponseItem) => r.status.message);
-                if (filteredFailures.length > 0) {
-                  console.log(`\n❌  VALIDATE   ${name} (${uid})`);
-                  console.log(`\u001b[1;31m${filteredFailures}\u001b[0m`);
+        sleep(2, () => {
```

Comment:
Should this be something like `await sleep(2);`?

Reply:
yes that is a good idea